addons/chemlights/XEH_preInit.sqf

```diff
@@ -0,0 +1,8 @@
+#include "script_component.hpp"
+
+ADDON = false;
+LOG(MSG_INIT);
```

Comment:
It's from CBA. We never use it. Outside of debug mode it does nothing. IDK if it even works in the ACE setup.